js/interface.js

```diff
@@ -691,6 +691,18 @@ function checkGroupErrors() {
 }
 
 /* ATTACH LISTENERS */
+
+$('[data-toggle="tooltip"]').tooltip({
+  title: function() {
+    var tooltipText = $(this).text();
+    if (tooltipText.length < 41) {
+      return;
+    }
+    return tooltipText;
+  },
+  delay: { "show": 500, "hide": 300 }
```

Comment:
double quotes are unnecessary here

Reply:
@squallstar That how bootstrap does it. Change to single quotes?

Reply:
@MaksymShokin you don't need quotes in Javascript object keys unless they have a dash or other non-common characters:

```js
{ foo: 'bar' }
```

```js
{ 'foo-bar': 'baz' }
```

Our code guidelines clearly state that:
1. Quotes should only be added when necessary
2. You should use single quote whenever possible

Reply:
@squallstar Fixed both.